site/docs/reference/Connectors/materialization-connectors/amazon-s3-csv.md

```diff
@@ -0,0 +1,96 @@
+
+# CSV Files in Amazon S3
+
+This connector materializes [delta updates](../../../concepts/materialization.md#delta-updates) of
+Flow collections into an S3 bucket in the CSV format.
+
+The delta updates are batched within Flow, converted to CSV files, and then pushed to the S3
+bucket at a time interval that you set.
+
+[`ghcr.io/estuary/materialize-s3-csv:dev`](https://ghcr.io/estuary/materialize-s3-csv:dev) provides
+the latest connector image. You can also follow the link in your browser to see past image versions.
+
+## Prerequisites
+
+To use this connector, you'll need:
+
+* An S3 bucket to write files to. See [this
+  guide](https://docs.aws.amazon.com/AmazonS3/latest/userguide/create-bucket-overview.html) for
+  instructions on setting up a new S3 bucket.
+* An AWS root or IAM user with access to the S3 bucket. For this user, you'll need the **access
+  key** and **secret access key**. See the [AWS
+  blog](https://aws.amazon.com/blogs/security/wheres-my-secret-access-key/) for help finding these
+  credentials.
+
+## Configuration
+
+Use the below properties to configure the materialization, which will direct one or more of your
+Flow collections to your bucket.
+
+### Properties
+
+#### Endpoint
```

Comment:
More questions about the connector (or maybe just parameters missing from this list):
- Does the connector support configuring a character used to denote a line break?
- Are quoted/unquoted fields supported?

Reply:
It doesn't support a character used to denote a line break. I'm not 100% sure what you mean by quoted/unquoted fields - fields are not quoted by default, but will be quoted with double quotes if required by [RFC 4180](https://www.rfc-editor.org/rfc/rfc4180.html).

Reply:
I added a bit to the intro section about how we are materializing to the `RFC-4180` definition of the CSV format, which is probably an important clarification. Also that we gzip the files.

Reply:
Gotcha, perfect!